### PR TITLE
Log docs

### PR DESCRIPTION
**Pull request by lewinkedrs**
This PR creates a central Logging page under the EKS folder. I chose to leave the log sections in the individual docs as well because it seems like a nicer experience if you are following all the directions to all see the log information at the end. This page will serve as a general overview of logging for all of the EKS patterns.

### CODE REVIEW

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-awsnative-observability.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ prometheus-node-exporter   Active   13m
 - On a same view, select 'EKS Pods', which provides insights overview of all the pods as shown below
 ![Container Insights - EKS Pods](../images/awsnative-container-insights-2.png)
 
-- Please see [Single New EKS Cluster AWS Mixed Observability Accelerator](../single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md#viewing-logs) for Logs.
+- Please see [Logging](../../logs.md) for information on viewing Logs.
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-awsnative-observability.md:105]: This change is missing in all other patterns. We want all patterns that logs to point to this page. like OSS, Mixed, AWS native in both single new and existing patterns

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@lewinkedrs Some feedback.

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-awsnative-observability.md (hunk 1 above)
Comment (lewinkedrs) [reply docs/patterns/existing-eks-observability-accelerators/existing-eks-awsnative-observability.md:105]: Linked to main log page in each of the logging subsections for each pattern.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability.md [hunk 2]
```diff
@@ -246,58 +246,7 @@ Grafana Operator and Flux always work together to synchronize your dashboards wi
...
-    You can add logs in your dashboards with logs panel types or time series
-    depending on your query results type.
-
-![logs-2](../images/logs-2.png)
-
-!!! warning
-    Querying CloudWatch logs will incur costs per GB scanned. Use small time
-    windows and limits in your queries. Checkout the CloudWatch
-    [pricing page](https://aws.amazon.com/cloudwatch/pricing/) for more infos.
+Please see [Logging](../../logs.md) for information on viewing Logs.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability.md:249]: Please be more specific say... refer to "Using CloudWatch Logs as data source in Grafana" section in [Logging]... as per the pattern.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-observability.md [hunk 3]
```diff
@@ -113,22 +113,7 @@ View the Performance Monitoring Dashboard:
...
-In the dropdown, select any of the logs that begin with "/aws/eks/single-new-eks-awsnative-observability-accelerator" and run a query.
-
-Example with "kubesystem" log group:
-
-![logs-query](../images/logs-insights1.png)
-
-Then you can view the results of your query:
-
-![logs-results](../images/logs-insights2.png)
+Please see [Logging](../../logs.md) for information on viewing Logs.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-observability.md:116]: Please be more specific. refer to section `Using CloudWatch Logs Insights to Query Logs` in the [Logging] page.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md [hunk 4]
```diff
@@ -106,22 +106,7 @@ View the graph of the selected metrics:
...
-In the dropdown, select any of the logs that begin with "/aws/eks/single-new-eks-mixed-observability-accelerator" and run a query.
-
-Example with "kubesystem" log group:
-
-![logs-query](../images/mixed-query.png)
-
-Then you can view the results of your query:
-
-![logs-results](../images/mixed-log-results.png)
+Please see [Logging](../../logs.md) for information on viewing Logs.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md:109]: Please be more specific. refer to section `Using CloudWatch Logs Insights to Query Logs` in the [Logging] page.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@lewinkedrs Great work, some minor feedback to verbiage and section pointers. Also in pages New Cluster OSS and Mixed pages, replace "Please see [Single New EKS Cluster AWS Mixed Observability Accelerator]" with right reference to right section in logging page like other pages you changed

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@lewinkedrs Looks great. My previous comment is still not addressed. You should add similar verbiage for existing mixed and OSS patterns vs referring to AWS native one

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**